modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/ManagementToolbar/index.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayManagementToolbar from '@clayui/management-toolbar';
+import {useModal} from '@clayui/modal';
+import {navigate} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+import {ModalEditExternalReferenceCode} from './ModalEditExternalReferenceCode';
+
+interface IProps {
+	backURL: string;
+	badgeClassName?: string;
+	badgeLabel?: string;
+	entityId: number;
+	externalReferenceCode: string;
+	externalReferenceCodeSaveURL: string;
+	getEntity: Function;
```

Comment:
hey @naraandrade, to follow a pattern I think we can rename all the props that will be of type Function to start with `on`, in this case `onGetEntity`.

Another thing is, do you think we can use a generic type to specify the return of this function? In this way we are going to improve this type making it more specific, as this function is related to getting an entity, we can use `() => Promise<T>` and in case of notification this Generic will be of type `NotificationTemplate` and in o case Object will be of type `ObjectDefinition`, what do you think?

Reply:
Great suggestions, I'll do it :)